### PR TITLE
crm_rma_location: migration V9

### PR DESCRIPTION
**Pull request by cyrilgdn**

### CODE REVIEW

**Comment by guewen** [comment]
:+1:

**Comment by mdietrichc2c** [comment]
:+1:

File: crm_rma_location/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* No issues are known yet
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/rma/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/rma/issues/new?body=module:%20crm_rma_location%0Aversion:%209.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (yvaucher) [inline crm_rma_location/README.rst]: You can remove that link. The template has been simplified

File: crm_rma_location/models/stock_warehouse.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            search_read([('warehouse_id', '!=', False),
+                         ('color', '!=', False)],
+                        ['color'], order='color')
+        for col in all_used_colors:
+            if col['color'] in available_colors:
+                available_colors.remove(col['color'])
+
+        return available_colors[0] if available_colors else 0
+
+    def create_sequence(self, warehouse_id, name, prefix, padding):
```
Comment (yvaucher) [inline crm_rma_location/models/stock_warehouse.py]: Why not an api.multi here ? Thus call it:

``` python
    warhouse.create_sequence(name, prefix, padding)
```

File: crm_rma_location/models/stock_warehouse.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            })
+
+    @api.multi
+    def create_locations_rma(self):
+        """
+        Create a RMA location for RMA movements that takes place when internal,
+        outgoing or incoming pickings are made from/to this location
+        """
+        location_obj = self.env['stock.location']
+        context_with_inactive = self.env.context.copy()
```
Comment (yvaucher) [inline crm_rma_location/models/stock_warehouse.py]: No need to copy the context. When using `with_context` the kwargs are overrides.

From `openerp/models.py`

``` python
            # current context is {'key1': True}
            r2 = records.with_context({}, key2=True)
            # -> r2._context is {'key2': True}
            r2 = records.with_context(key2=True)
            # -> r2._context is {'key1': True, 'key2': True}
```

File: crm_rma_location/tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline crm_rma_location/tests/__init__.py]: Header to change (in an `__init__.py` file without logic you can even completely remove it)

File: crm_rma_location/tests/test_location_rma.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        """
+
+        warehouse_id = self.warehouse.create({
+            'name': 'BrandNew WH',
+            'code': 'NEWWH'
+        })
+
+        self.assertTrue(warehouse_id.rma_in_type_id and
+                        warehouse_id.rma_out_type_id.code and
+                        warehouse_id.rma_int_type_id.code)
```
Comment (yvaucher) [inline crm_rma_location/tests/test_location_rma.py]: If `warehouse_id.rma_in_type_id` is an empty record set `warehouse_id.rma_in_type.code` will return `False`

File: crm_rma_location/views/stock_warehouse.xml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```
Comment (yvaucher) [inline crm_rma_location/views/stock_warehouse.xml]: You can use `<odoo>`

File: crm_rma_location/tests/test_location_rma.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        """
+        Check if picking types were created
+        """
+
+        warehouse_id = self.warehouse.create({
+            'name': 'BrandNew WH',
+            'code': 'NEWWH'
+        })
+
+        self.assertTrue(warehouse_id.rma_in_type_id and
```
Comment (yvaucher) [inline crm_rma_location/tests/test_location_rma.py:24]: Don't you want to check code too here ?

**Comment by yvaucher** [comment]
Few remarks otherwise LGTM

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**